pkgs/os-specific/linux/util-linux/default.nix

```diff
@@ -1,10 +1,11 @@
 { lib, stdenv, fetchurl, pkg-config, zlib, shadow, libcap_ng
-, ncurses ? null, pam, systemd ? null
+, ncurses ? null, pam, withPAM ? true, systemd ? null
 , nlsSupport ? true
 , audit ? null
 }:
 
-assert stdenv.hostPlatform.isStatic -> audit != null;
+assert withPAM -> pam != null;
```

Comment:
Is that package used in bootstrapping? If not the input packages should not have ``? null`` and we don't need to check this.